Open Judge System/Services/OJS.Services/Data/Submissions/SubmissionsDataService.cs

```diff
@@ -1,4 +1,7 @@
-﻿namespace OJS.Services.Data.Submissions
+﻿using System.Data.Entity;
+using MissingFeatures;
```

Comment:
move usings in a namespace or move the usings below which are in the namespace of it to be consistent